internal/analytics/tracks.go

```diff
@@ -66,3 +66,35 @@ func (t *segmentRedeployViaWebhookTrack) getEvent() SegmentEvent {
 func (t *segmentRedeployViaWebhookTrack) getProperties() segment.Properties {
 	return segment.NewProperties().Set("repository", t.repository)
 }
+
+type segmentNewClusterEventTrack struct {
+	userId    string
+	projId string
+	clusterName string
+	clusterType string // EKS, DOKS, or GKE
+	eventType string // connected, provisioned, or destroyed
+}
+
+// Constructor for track of type "New Cluster Event"
```

Comment:
```suggestion
// CreateSegmentNewClusterEvent tracks whenever...
```

Exported methods should have comments which begin with the method name.

Reply:
As I ended up messing this up on the rest of the package, we should also document this on a backend style guide. Idk if it's okay to change only this comment for now and after that, update the rest of the package comments on another PR or do all of them on the new PR and leave this one as it is for now.